app/lib/features/chat/providers/notifiers/chat_input_notifier.dart

```diff
@@ -17,11 +17,11 @@ class ChatInputNotifier extends StateNotifier<ChatInputState> {
     state = state.copyWith(attachmentVisible: !state.attachmentVisible);
   }
 
-  void emojiRowVisible() =>
-      state = state.copyWith(emojiRowVisible: !state.emojiRowVisible);
+  void emojiRowVisible(bool value) =>
+      state = state.copyWith(emojiRowVisible: value);
```

Comment:
Good. I prefer these hard bool-setters rather than toggle-switches ... it is a lot more clear from the UI point of view, what is expected to happen (and other calls in between do not mess with that state).